src/WiFi101OTA.h

```diff
@@ -21,13 +21,32 @@
 
 #include <Arduino.h>
 
-#include "WiFi101.h"
+#if defined(ARDUINO_SAMD_ZERO)
+  #include "WiFi101.h"
+  #define BOARD "arduino_zero_edbg"
+#elif defined(ARDUINO_SAMD_MKR1000)
+  #include "WiFi101.h"
+  #define BOARD "mkr1000"
+#elif defined(ARDUINO_SAMD_MKRZERO)
+  #include "WiFi101.h"
+  #define BOARD "mkrzero"
+#elif defined(ARDUINO_SAMD_MKRWIFI1010)
```

Comment:
Aren't the Nano 33 IoT and MKR Vidor 4000 also supported?
```suggestion
#elif defined(ARDUINO_SAMD_MKRWIFI1010) || defined(ARDUINO_SAMD_NANO_33_IOT) || defined(ARDUINO_SAMD_MKRVIDOR4000)
```

Reply:
Yup, but they need to have a separate entry each to get recognized (by name) by the network discoverer

Reply:
Fixed by https://github.com/arduino-libraries/WiFi101OTA/pull/15/commits/104617302c02ec88acd0e387dc88adb1e22c9162